### ISSUE TITLE
Subphylum Crustacea interpreted as (doubtful) genus Crustacea

### ISSUE THREAD

**Issue by dagendresen** [gbif/portal-feedback#3382]
When publishing a new [dataset on plankton](https://doi.org/10.15468/c2m9zk) the [occurrence records](https://www.gbif.org/occurrence/search?dataset_key=e697326b-48d3-44ee-8b8d-0c9ee26c16d7&taxon_key=10996236) with scientificName = **subphylum Crustacea** are interpreted as a (flagged doubtful) **genus Crustacea**.

See [subphylum Crustacea, urn:lsid:marinespecies.org:taxname:1066](http://www.marinespecies.org/aphia.php?p=taxdetails&id=1066) in WORMS.
And the taxon record for Crustacea from WORMS in the GBIF backbone, https://www.gbif.org/species/155486131

<img width="898" alt="Screenshot 2021-04-22 at 11 13 03" src="https://user-images.githubusercontent.com/4330242/115690759-a6c2c380-a35d-11eb-9ad5-68d2b5f5f1d2.png">

**Comment by dagendresen** [comment]
See also https://github.com/gbif/pipelines/issues/217

**Comment by ManonGros** [comment]
That's probably more a question for @mdoering but from what I understand:
1. the subphylum Crustacea is not in the GBIF backbone (the only taxon ranks integrated in the backbone are `Kingdom`, `Phylum`, `Class`, `Order`, `Family`, `Genus`, `Species`, `Subspecies` and `unranked` for some OTUs: https://data-blog.gbif.org/post/gbif-backbone-taxonomy/)
2. So Crustacea is matched to the only thing we have in the backbone: https://www.gbif.org/species/10996236 which is a Hymenoptera genus and is labelled as `doubtful` because it doesn't have any associated species.

So for now the solution would be to use one of the ranks available in the backbone.

**Comment by dagendresen** [comment]
I understand so that these are nauplii (Crustacean larva) and very hard to identify to a lower taxon rank, thus all knowable taxon information is included with the occurrences.

**Comment by rukayaj** [comment]
@ManonGros Are you suggesting we go up one level to `Phylum`?

**Comment by dagendresen** [comment]
I see that the dataset already includes taxon information phylum = Arthropoda. Do you mean to reduce information value by removing subphylum all together? Would it maybe be possible to explore the erroneous GBIF interpretation, ignoring information such as rank provided, for the occurrences could be explored first?? :-)

Apropos see also the taxon **Crustacea Brünnich, 1772** from the Norwegian Species Cehecklist https://www.gbif.org/species/167991571

I believe that many more similar issues will arise when mobilizing more marine datasets ;-)

**Comment by mdoering** [comment]
Yes, GBIF must be able to understand Crustacea and link to somewhere sensible without the need to dumn down the supplied data. We can maintain a small manual lookup map for important cases that often go wrong like this one. 

Would phylum Arthropoda be the best place to link those records to?
COL accepts now basically any rank and does not restrict higher taxa to Linnean ranks only. Hopefully in the not too distant future GBIF will allow the same. There are more serious problems on the horizon such as Aves being a subphylum. [Vertebrates](https://github.com/gbif/portal-feedback/issues/127) also do not exist right now in the backbone, again a subphylum. @timrobertson100 maybe we should quickly add dwc:subphylum to april fools? Seems this is a really important one for occurrences.

**Comment by timrobertson100** [comment]
> maybe we should quickly add dwc:subphylum to [april fools](https://github.com/tdwg/dwc/milestone/14)

Seems sensible. It will also mean taxa at these ranks will need to be added in the backbone

**Comment by dagendresen** [comment]
Ping - Norwegian marine data publishers are asking for status on this issue :-)
